index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href='styles.css'>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Challenge</title>
+  </head>
+  <body>
+    <h1>Adam Smith</h1>
+    <p id="bio">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.<a href="https://www.linkedin.com">linkedin</a> Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+    <h2>Education history</h2>
```

Comment:
Start your project with the header part, so it will be better semantically.

Reply:
- [ ] **Good job on using the github inline comments 👍** however, it would be better to phrase the comments politely, as suggestions instead of orders. 😎😎

For example:

This is a polite suggestion: "I think it would be better to Start your project with the header part, ...."

This reads as an order: "Start your project with header..." 😂